int32 Level
string Message
string Date
